lemur/elasticsearch_logger.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+import logging
+from logging import getLogger, Handler
+
+from elasticsearch import Elasticsearch
+from pytz import timezone
+
+
+class ESHandler(Handler):
+    def __init__(self, es, index, elasticsearch_logging_level="error"):
+        super().__init__()
+        self.es = Elasticsearch(es)
+        self.index = index
+
+        if elasticsearch_logging_level == "info":
+            level = logging.INFO
+        elif elasticsearch_logging_level == "critical":
+            level = logging.CRITICAL
+        elif elasticsearch_logging_level == "error":
+            level = logging.ERROR
+        elif elasticsearch_logging_level == "warning":
+            level = logging.WARNING
+        elif elasticsearch_logging_level == "debug":
+            level = logging.DEBUG
+
+        getLogger("elasticsearch").setLevel(level)
+        getLogger("elasticsearch.trace").setLevel(level)
+        getLogger("urllib3").setLevel(level)
+
+    def emit(self, record):
+        record.eventTime = datetime.datetime.now(timezone('US/Pacific')).isoformat()
+        log_entry = self.format(record)
+
+        # Push action to ES
+        return self.es.index(index=self.index, doc_type="python_log", body=log_entry)
```

Comment:
Wrap this in `try/except` to prevent breakage on logging failure.

Reply:
I think the concern is that, the logic isn't necessary to have in Lemur at all. Write to a file (which is default/standard) and then let some other process figure out buffering etc., For instance we'd wouldn't want to block/delay a request simply because ES is taking a while to respond.